src/endian_reader.rs

```diff
@@ -92,9 +92,21 @@ pub type EndianArcSlice<Endian> = EndianReader<Endian, Arc<[u8]>>;
 ///     }
 /// }
 ///
+/// /// A type that represents a shared `mmap`ed file.
+/// #[derive(Debug, Clone)]
+/// pub struct ArcMmapFile(Arc<MmapFile>);
```

Comment:
Right, and this *is* the thing that should be clone, since it is just bumping a refcount.